test/gold/points-to_malloc-context.clang.-g-grecord-gcc-switches-O1-O0-fno-discard-value-names.1-callsite.allocation_size.golden.csv

```diff
@@ -1,4 +1,3 @@
-*global_alloc@global_obj	8
```

Comment:
Triage: These changes are suspicious and should be reviewed before merging. Looks like the same thing as above.

Reply:
<details>
<summary>Comparison script</summary>

```python
    from pathlib import Path
    from subprocess import PIPE, check_output, run
    from sys import stdin, stdout, stderr
    from textwrap import indent

    BASH = """
    sudo rm -rf 10 11
    docker_here() {
        docker run --rm -it --mount type=bind,src=$PWD,target=/work --workdir /work "${@}"
    }
    docker_here cclyzer-dev-llvm-11 clang-11 -O1 -O1 -fno-discard-value-names -S -emit-llvm test/c/points-to_malloc-context.c -o 11.ll
    docker pull ghcr.io/galoisinc/cclyzerpp-dev:main
    docker tag ghcr.io/galoisinc/cclyzerpp-dev:main cclyzer-dev
    docker_here cclyzer-dev clang-10 -O1 -O1 -fno-discard-value-names -S -emit-llvm test/c/points-to_malloc-context.c -o 10.ll
    docker_here cclyzer-dev-llvm-11 cmake -G Ninja -S . -B build -DLLVM_VERSION=11 -DCLANG_VERSION=11
    docker_here cclyzer-dev-llvm-11 cmake --build build -j $(nproc) --target factgen-exe
    docker_here cclyzer-dev-llvm-11 ./build/factgen-exe 11.ll --out-dir 11
    sudo chown -R $(whoami):$(whoami) 11
    for f in 11/*.gz; do gunzip "${f}"; done
    for f in 11/*.csv; do sort "${f}" > "${f}.sorted"; rm "${f}"; done
    docker_here cclyzer-dev cmake -G Ninja -S . -B build-10 -DLLVM_VERSION=10 -DCLANG_VERSION=10
    docker_here cclyzer-dev cmake --build build-10 -j $(nproc) --target factgen-exe
    docker_here cclyzer-dev ./build/factgen-exe 10.ll --out-dir 10
    sudo chown -R $(whoami):$(whoami) 10
    for f in 10/*.gz; do gunzip "${f}"; done
    for f in 10/*.csv; do sort "${f}" > "${f}.sorted"; rm "${f}"; done
    pushd 10
    sed_recurse() { ag -g '.*' -0 | xargs -0 sed -E -i "$@"; }
    sed_recurse 's|10\.ll|11\.ll|g'
    popd
    for f in 10/*.csv.sorted; do diff -u "${f}" 11/$(basename ${f}); done
    """

    TEMPLATE = """
    <details>
    <summary>Comparison script</summary>

    ```python
    {script}
    ```

    </details>
    <details>
    <summary>10.ll</summary>

    ```llvm
    {ten}
    ```

    </details>

    <details>
    <summary>11.ll</summary>

    ```llvm
    {eleven}
    ```

    </details>

    <details>
    <summary>LLVM diff</summary>

    ```diff
    {lldiff}
    ```

    </details>

    <details>
    <summary>Relation diff</summary>

    ```diff
    {reldiff}
    ```

    </details>
    """

    if __name__ == "__main__":
        Path("script.sh").write_text(BASH)
        out = check_output(["bash", "script.sh"], stdin=stdin, stderr=stderr)
        print(TEMPLATE.format(
            script=indent(Path(__file__).read_text(), "    "),
            ten=Path("10.ll").read_text(),
            eleven=Path("11.ll").read_text(),
            lldiff=run(["diff", "-u", "10.ll", "11.ll"], stdout=PIPE).stdout.decode('utf-8'),
            reldiff=out.decode('utf-8'),
        ))

```

</details>
<details>
<summary>10.ll</summary>

```llvm
; ModuleID = 'test/c/points-to_malloc-context.c'
source_filename = "test/c/points-to_malloc-context.c"
target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-pc-linux-gnu"

%struct.obj = type { i32, i32 }

@global_obj = common dso_local global %struct.obj zeroinitializer, align 4

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local %struct.obj* @id(%struct.obj* readnone returned %x) local_unnamed_addr #0 {
entry:
  ret %struct.obj* %x
}

; Function Attrs: nofree nounwind uwtable
define dso_local %struct.obj* @fun1() local_unnamed_addr #1 {
entry:
  %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
  %0 = bitcast i8* %call to %struct.obj*
  %call1 = call %struct.obj* @id(%struct.obj* %0)
  ret %struct.obj* %call1
}

; Function Attrs: nofree nounwind
declare dso_local noalias i8* @malloc(i64) local_unnamed_addr #2

; Function Attrs: nofree nounwind uwtable
define dso_local %struct.obj* @fun2() local_unnamed_addr #1 {
entry:
  %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
  %0 = bitcast i8* %call to %struct.obj*
  %call1 = call %struct.obj* @id(%struct.obj* %0)
  ret %struct.obj* %call1
}

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local nonnull %struct.obj* @fun3() local_unnamed_addr #0 {
entry:
  %call = call %struct.obj* @id(%struct.obj* nonnull @global_obj)
  ret %struct.obj* %call
}

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local nonnull %struct.obj* @fun4() local_unnamed_addr #0 {
entry:
  %call = call %struct.obj* @id(%struct.obj* nonnull @global_obj)
  ret %struct.obj* %call
}

; Function Attrs: nofree nounwind uwtable
define dso_local i32 @main() local_unnamed_addr #1 {
entry:
  %call = call %struct.obj* @fun1()
  %call1 = call %struct.obj* @fun2()
  %call2 = call %struct.obj* @fun3()
  %call3 = call %struct.obj* @fun4()
  %field1 = getelementptr inbounds %struct.obj, %struct.obj* %call, i64 0, i32 0
  %0 = load i32, i32* %field1, align 4, !tbaa !2
  %field14 = getelementptr inbounds %struct.obj, %struct.obj* %call1, i64 0, i32 0
  %1 = load i32, i32* %field14, align 4, !tbaa !2
  %add = add nsw i32 %1, %0
  %field15 = getelementptr inbounds %struct.obj, %struct.obj* %call2, i64 0, i32 0
  %2 = load i32, i32* %field15, align 4, !tbaa !2
  %add6 = add nsw i32 %add, %2
  %field17 = getelementptr inbounds %struct.obj, %struct.obj* %call3, i64 0, i32 0
  %3 = load i32, i32* %field17, align 4, !tbaa !2
  %add8 = add nsw i32 %add6, %3
  ret i32 %add8
}

attributes #0 = { norecurse nounwind readnone uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #1 = { nofree nounwind uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #2 = { nofree nounwind "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #3 = { nounwind }

!llvm.module.flags = !{!0}
!llvm.ident = !{!1}

!0 = !{i32 1, !"wchar_size", i32 4}
!1 = !{!"clang version 10.0.0-4ubuntu1 "}
!2 = !{!3, !4, i64 0}
!3 = !{!"obj", !4, i64 0, !4, i64 4}
!4 = !{!"int", !5, i64 0}
!5 = !{!"omnipotent char", !6, i64 0}
!6 = !{!"Simple C/C++ TBAA"}

```

</details>

<details>
<summary>11.ll</summary>

```llvm
; ModuleID = 'test/c/points-to_malloc-context.c'
source_filename = "test/c/points-to_malloc-context.c"
target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-pc-linux-gnu"

%struct.obj = type { i32, i32 }

@global_obj = dso_local global %struct.obj zeroinitializer, align 4

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local %struct.obj* @id(%struct.obj* readnone returned %x) local_unnamed_addr #0 {
entry:
  ret %struct.obj* %x
}

; Function Attrs: nofree nounwind uwtable
define dso_local %struct.obj* @fun1() local_unnamed_addr #1 {
entry:
  %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
  %0 = bitcast i8* %call to %struct.obj*
  ret %struct.obj* %0
}

; Function Attrs: nofree nounwind
declare dso_local noalias i8* @malloc(i64) local_unnamed_addr #2

; Function Attrs: nofree nounwind uwtable
define dso_local %struct.obj* @fun2() local_unnamed_addr #1 {
entry:
  %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
  %0 = bitcast i8* %call to %struct.obj*
  ret %struct.obj* %0
}

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local nonnull %struct.obj* @fun3() local_unnamed_addr #0 {
entry:
  ret %struct.obj* @global_obj
}

; Function Attrs: norecurse nounwind readnone uwtable
define dso_local nonnull %struct.obj* @fun4() local_unnamed_addr #0 {
entry:
  ret %struct.obj* @global_obj
}

; Function Attrs: nofree nounwind uwtable
define dso_local i32 @main() local_unnamed_addr #1 {
entry:
  %call = call %struct.obj* @fun1()
  %call1 = call %struct.obj* @fun2()
  %call2 = call %struct.obj* @fun3()
  %call3 = call %struct.obj* @fun4()
  %field1 = getelementptr inbounds %struct.obj, %struct.obj* %call, i64 0, i32 0
  %0 = load i32, i32* %field1, align 4, !tbaa !2
  %field14 = getelementptr inbounds %struct.obj, %struct.obj* %call1, i64 0, i32 0
  %1 = load i32, i32* %field14, align 4, !tbaa !2
  %add = add nsw i32 %1, %0
  %field15 = getelementptr inbounds %struct.obj, %struct.obj* %call2, i64 0, i32 0
  %2 = load i32, i32* %field15, align 4, !tbaa !2
  %add6 = add nsw i32 %add, %2
  %field17 = getelementptr inbounds %struct.obj, %struct.obj* %call3, i64 0, i32 0
  %3 = load i32, i32* %field17, align 4, !tbaa !2
  %add8 = add nsw i32 %add6, %3
  ret i32 %add8
}

attributes #0 = { norecurse nounwind readnone uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #1 = { nofree nounwind uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #2 = { nofree nounwind "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
attributes #3 = { nounwind }

!llvm.module.flags = !{!0}
!llvm.ident = !{!1}

!0 = !{i32 1, !"wchar_size", i32 4}
!1 = !{!"Ubuntu clang version 11.0.0-2~ubuntu20.04.1"}
!2 = !{!3, !4, i64 0}
!3 = !{!"obj", !4, i64 0, !4, i64 4}
!4 = !{!"int", !5, i64 0}
!5 = !{!"omnipotent char", !6, i64 0}
!6 = !{!"Simple C/C++ TBAA"}

```

</details>

<details>
<summary>LLVM diff</summary>

```diff
--- 10.ll	2022-10-25 15:45:33.478545553 -0400
+++ 11.ll	2022-10-25 15:45:32.730539984 -0400
@@ -5,7 +5,7 @@
 
 %struct.obj = type { i32, i32 }
 
-@global_obj = common dso_local global %struct.obj zeroinitializer, align 4
+@global_obj = dso_local global %struct.obj zeroinitializer, align 4
 
 ; Function Attrs: norecurse nounwind readnone uwtable
 define dso_local %struct.obj* @id(%struct.obj* readnone returned %x) local_unnamed_addr #0 {
@@ -18,8 +18,7 @@
 entry:
   %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
   %0 = bitcast i8* %call to %struct.obj*
-  %call1 = call %struct.obj* @id(%struct.obj* %0)
-  ret %struct.obj* %call1
+  ret %struct.obj* %0
 }
 
 ; Function Attrs: nofree nounwind
@@ -30,22 +29,19 @@
 entry:
   %call = call noalias dereferenceable_or_null(8) i8* @malloc(i64 8) #3
   %0 = bitcast i8* %call to %struct.obj*
-  %call1 = call %struct.obj* @id(%struct.obj* %0)
-  ret %struct.obj* %call1
+  ret %struct.obj* %0
 }
 
 ; Function Attrs: norecurse nounwind readnone uwtable
 define dso_local nonnull %struct.obj* @fun3() local_unnamed_addr #0 {
 entry:
-  %call = call %struct.obj* @id(%struct.obj* nonnull @global_obj)
-  ret %struct.obj* %call
+  ret %struct.obj* @global_obj
 }
 
 ; Function Attrs: norecurse nounwind readnone uwtable
 define dso_local nonnull %struct.obj* @fun4() local_unnamed_addr #0 {
 entry:
-  %call = call %struct.obj* @id(%struct.obj* nonnull @global_obj)
-  ret %struct.obj* %call
+  ret %struct.obj* @global_obj
 }
 
 ; Function Attrs: nofree nounwind uwtable
@@ -69,16 +65,16 @@
   ret i32 %add8
 }
 
-attributes #0 = { norecurse nounwind readnone uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
-attributes #1 = { nofree nounwind uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
-attributes #2 = { nofree nounwind "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
+attributes #0 = { norecurse nounwind readnone uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
+attributes #1 = { nofree nounwind uwtable "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "min-legal-vector-width"="0" "no-infs-fp-math"="false" "no-jump-tables"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
+attributes #2 = { nofree nounwind "correctly-rounded-divide-sqrt-fp-math"="false" "disable-tail-calls"="false" "frame-pointer"="none" "less-precise-fpmad"="false" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false" "no-trapping-math"="true" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+cx8,+fxsr,+mmx,+sse,+sse2,+x87" "unsafe-fp-math"="false" "use-soft-float"="false" }
 attributes #3 = { nounwind }
 
 !llvm.module.flags = !{!0}
 !llvm.ident = !{!1}
 
 !0 = !{i32 1, !"wchar_size", i32 4}
-!1 = !{!"clang version 10.0.0-4ubuntu1 "}
+!1 = !{!"Ubuntu clang version 11.0.0-2~ubuntu20.04.1"}
 !2 = !{!3, !4, i64 0}
 !3 = !{!"obj", !4, i64 0, !4, i64 4}
 !4 = !{!"int", !5, i64 0}

```

</details>

<details>
<summary>Relation diff</summary>

```diff
main: Pulling from galoisinc/cclyzerpp-dev
Digest: sha256:94c9f10add529715192f36dfc5a38d5cac8bbab86b21eb210e54ffb861c09d63
Status: Image is up to date for ghcr.io/galoisinc/cclyzerpp-dev:main
ghcr.io/galoisinc/cclyzerpp-dev:main
-- Found LLVM 11.0.0
-- Using LLVMConfig.cmake in: /usr/lib/llvm-11/cmake
-- Configuring done
-- Generating done
-- Build files have been written to: /work/build
ninja: no work to do.
-- Found LLVM 10.0.0
-- Using LLVMConfig.cmake in: /usr/lib/llvm-10/cmake
-- Configuring done
-- Generating done
-- Build files have been written to: /work/build-10
ninja: no work to do.
~/code/cclyzer/10 ~/code/cclyzer
~/code/cclyzer
--- 10/call_instr_arg.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/call_instr_arg.csv.sorted	2022-10-25 15:45:35.311559193 -0400
@@ -1,6 +1,2 @@
 </work/11.ll>:fun1:0	0	</work/11.ll>:fun1:0:1:8
-</work/11.ll>:fun1:2	0	</work/11.ll>:fun1:%0
 </work/11.ll>:fun2:0	0	</work/11.ll>:fun2:0:1:8
-</work/11.ll>:fun2:2	0	</work/11.ll>:fun2:%0
-</work/11.ll>:fun3:0	0	</work/11.ll>:fun3:0:1:@global_obj
-</work/11.ll>:fun4:0	0	</work/11.ll>:fun4:0:1:@global_obj
--- 10/call_instr.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/call_instr.csv.sorted	2022-10-25 15:45:35.313559208 -0400
@@ -1,9 +1,5 @@
 </work/11.ll>:fun1:0
-</work/11.ll>:fun1:2
 </work/11.ll>:fun2:0
-</work/11.ll>:fun2:2
-</work/11.ll>:fun3:0
-</work/11.ll>:fun4:0
 </work/11.ll>:main:0
 </work/11.ll>:main:1
 </work/11.ll>:main:2
--- 10/call_instr_func_operand.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/call_instr_func_operand.csv.sorted	2022-10-25 15:45:35.316559231 -0400
@@ -1,9 +1,5 @@
 </work/11.ll>:fun1:0	</work/11.ll>:fun1:0:0:@malloc
-</work/11.ll>:fun1:2	</work/11.ll>:fun1:2:0:@id
 </work/11.ll>:fun2:0	</work/11.ll>:fun2:0:0:@malloc
-</work/11.ll>:fun2:2	</work/11.ll>:fun2:2:0:@id
-</work/11.ll>:fun3:0	</work/11.ll>:fun3:0:0:@id
-</work/11.ll>:fun4:0	</work/11.ll>:fun4:0:0:@id
 </work/11.ll>:main:0	</work/11.ll>:main:0:0:@fun1
 </work/11.ll>:main:1	</work/11.ll>:main:1:0:@fun2
 </work/11.ll>:main:2	</work/11.ll>:main:2:0:@fun3
--- 10/call_instr_param_attr.csv.sorted	2022-10-25 15:45:37.818577835 -0400
+++ 11/call_instr_param_attr.csv.sorted	2022-10-25 15:45:35.317559238 -0400
@@ -1,2 +0,0 @@
-</work/11.ll>:fun3:0	0	nonnull
-</work/11.ll>:fun4:0	0	nonnull
--- 10/constant.csv.sorted	2022-10-25 15:45:37.812577790 -0400
+++ 11/constant.csv.sorted	2022-10-25 15:45:35.335559371 -0400
@@ -1,14 +1,10 @@
 </work/11.ll>:0:zeroinitializer
 </work/11.ll>:fun1:0:0:@malloc
 </work/11.ll>:fun1:0:1:8
-</work/11.ll>:fun1:2:0:@id
 </work/11.ll>:fun2:0:0:@malloc
 </work/11.ll>:fun2:0:1:8
-</work/11.ll>:fun2:2:0:@id
-</work/11.ll>:fun3:0:0:@id
-</work/11.ll>:fun3:0:1:@global_obj
-</work/11.ll>:fun4:0:0:@id
-</work/11.ll>:fun4:0:1:@global_obj
+</work/11.ll>:fun3:0:0:@global_obj
+</work/11.ll>:fun4:0:0:@global_obj
 </work/11.ll>:main:0:0:@fun1
 </work/11.ll>:main:1:0:@fun2
 </work/11.ll>:main:12:0:0
--- 10/constant_hashes_to.csv.sorted	2022-10-25 15:45:37.814577805 -0400
+++ 11/constant_hashes_to.csv.sorted	2022-10-25 15:45:35.338559394 -0400
@@ -1,14 +1,10 @@
 </work/11.ll>:0:zeroinitializer	8555577089736398497
 </work/11.ll>:fun1:0:0:@malloc	10812519832347109215
 </work/11.ll>:fun1:0:1:8	13429359871425303597
-</work/11.ll>:fun1:2:0:@id	4475862324401889717
 </work/11.ll>:fun2:0:0:@malloc	10812519832347109215
 </work/11.ll>:fun2:0:1:8	13429359871425303597
-</work/11.ll>:fun2:2:0:@id	4475862324401889717
-</work/11.ll>:fun3:0:0:@id	4475862324401889717
-</work/11.ll>:fun3:0:1:@global_obj	1798808042309292186
-</work/11.ll>:fun4:0:0:@id	4475862324401889717
-</work/11.ll>:fun4:0:1:@global_obj	1798808042309292186
+</work/11.ll>:fun3:0:0:@global_obj	1798808042309292186
+</work/11.ll>:fun4:0:0:@global_obj	1798808042309292186
 </work/11.ll>:main:0:0:@fun1	6087127537543525555
 </work/11.ll>:main:1:0:@fun2	10199790805540868013
 </work/11.ll>:main:12:0:0	4490122437319657922
--- 10/constant_has_type.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/constant_has_type.csv.sorted	2022-10-25 15:45:35.339559402 -0400
@@ -1,14 +1,10 @@
 </work/11.ll>:0:zeroinitializer	%struct.obj
 </work/11.ll>:fun1:0:0:@malloc	i8* (i64)*
 </work/11.ll>:fun1:0:1:8	i64
-</work/11.ll>:fun1:2:0:@id	%struct.obj* (%struct.obj*)*
 </work/11.ll>:fun2:0:0:@malloc	i8* (i64)*
 </work/11.ll>:fun2:0:1:8	i64
-</work/11.ll>:fun2:2:0:@id	%struct.obj* (%struct.obj*)*
-</work/11.ll>:fun3:0:0:@id	%struct.obj* (%struct.obj*)*
-</work/11.ll>:fun3:0:1:@global_obj	%struct.obj*
-</work/11.ll>:fun4:0:0:@id	%struct.obj* (%struct.obj*)*
-</work/11.ll>:fun4:0:1:@global_obj	%struct.obj*
+</work/11.ll>:fun3:0:0:@global_obj	%struct.obj*
+</work/11.ll>:fun4:0:0:@global_obj	%struct.obj*
 </work/11.ll>:main:0:0:@fun1	%struct.obj* ()*
 </work/11.ll>:main:1:0:@fun2	%struct.obj* ()*
 </work/11.ll>:main:12:0:0	i64
--- 10/constant_has_value.csv.sorted	2022-10-25 15:45:37.816577820 -0400
+++ 11/constant_has_value.csv.sorted	2022-10-25 15:45:35.340559409 -0400
@@ -1,14 +1,10 @@
 </work/11.ll>:0:zeroinitializer	zeroinitializer
 </work/11.ll>:fun1:0:0:@malloc	@malloc
 </work/11.ll>:fun1:0:1:8	8
-</work/11.ll>:fun1:2:0:@id	@id
 </work/11.ll>:fun2:0:0:@malloc	@malloc
 </work/11.ll>:fun2:0:1:8	8
-</work/11.ll>:fun2:2:0:@id	@id
-</work/11.ll>:fun3:0:0:@id	@id
-</work/11.ll>:fun3:0:1:@global_obj	@global_obj
-</work/11.ll>:fun4:0:0:@id	@id
-</work/11.ll>:fun4:0:1:@global_obj	@global_obj
+</work/11.ll>:fun3:0:0:@global_obj	@global_obj
+</work/11.ll>:fun4:0:0:@global_obj	@global_obj
 </work/11.ll>:main:0:0:@fun1	@fun1
 </work/11.ll>:main:1:0:@fun2	@fun2
 </work/11.ll>:main:12:0:0	0
--- 10/constant_in_func_name.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/constant_in_func_name.csv.sorted	2022-10-25 15:45:35.342559424 -0400
@@ -1,13 +1,9 @@
 </work/11.ll>:fun1:0:0:@malloc	@fun1
 </work/11.ll>:fun1:0:1:8	@fun1
-</work/11.ll>:fun1:2:0:@id	@fun1
 </work/11.ll>:fun2:0:0:@malloc	@fun2
 </work/11.ll>:fun2:0:1:8	@fun2
-</work/11.ll>:fun2:2:0:@id	@fun2
-</work/11.ll>:fun3:0:0:@id	@fun3
-</work/11.ll>:fun3:0:1:@global_obj	@fun3
-</work/11.ll>:fun4:0:0:@id	@fun4
-</work/11.ll>:fun4:0:1:@global_obj	@fun4
+</work/11.ll>:fun3:0:0:@global_obj	@fun3
+</work/11.ll>:fun4:0:0:@global_obj	@fun4
 </work/11.ll>:main:0:0:@fun1	@main
 </work/11.ll>:main:1:0:@fun2	@main
 </work/11.ll>:main:12:0:0	@main
--- 10/func_constant.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/func_constant.csv.sorted	2022-10-25 15:45:35.412559944 -0400
@@ -1,9 +1,5 @@
 </work/11.ll>:fun1:0:0:@malloc
-</work/11.ll>:fun1:2:0:@id
 </work/11.ll>:fun2:0:0:@malloc
-</work/11.ll>:fun2:2:0:@id
-</work/11.ll>:fun3:0:0:@id
-</work/11.ll>:fun4:0:0:@id
 </work/11.ll>:main:0:0:@fun1
 </work/11.ll>:main:1:0:@fun2
 </work/11.ll>:main:2:0:@fun3
--- 10/func_constant_fn_name.csv.sorted	2022-10-25 15:45:37.815577813 -0400
+++ 11/func_constant_fn_name.csv.sorted	2022-10-25 15:45:35.413559952 -0400
@@ -1,9 +1,5 @@
 </work/11.ll>:fun1:0:0:@malloc	@malloc
-</work/11.ll>:fun1:2:0:@id	@id
 </work/11.ll>:fun2:0:0:@malloc	@malloc
-</work/11.ll>:fun2:2:0:@id	@id
-</work/11.ll>:fun3:0:0:@id	@id
-</work/11.ll>:fun4:0:0:@id	@id
 </work/11.ll>:main:0:0:@fun1	@fun1
 </work/11.ll>:main:1:0:@fun2	@fun2
 </work/11.ll>:main:2:0:@fun3	@fun3
--- 10/func_func_attr.csv.sorted	2022-10-25 15:45:37.813577797 -0400
+++ 11/func_func_attr.csv.sorted	2022-10-25 15:45:35.417559982 -0400
@@ -8,7 +8,7 @@
 </work/11.ll>:fun1	no-jump-tables=false
 </work/11.ll>:fun1	no-nans-fp-math=false
 </work/11.ll>:fun1	no-signed-zeros-fp-math=false
-</work/11.ll>:fun1	no-trapping-math=false
+</work/11.ll>:fun1	no-trapping-math=true
 </work/11.ll>:fun1	nounwind
 </work/11.ll>:fun1	stack-protector-buffer-size=8
 </work/11.ll>:fun1	target-cpu=x86-64
@@ -26,7 +26,7 @@
 </work/11.ll>:fun2	no-jump-tables=false
 </work/11.ll>:fun2	no-nans-fp-math=false
 </work/11.ll>:fun2	no-signed-zeros-fp-math=false
-</work/11.ll>:fun2	no-trapping-math=false
+</work/11.ll>:fun2	no-trapping-math=true
 </work/11.ll>:fun2	nounwind
 </work/11.ll>:fun2	stack-protector-buffer-size=8
 </work/11.ll>:fun2	target-cpu=x86-64
@@ -44,7 +44,7 @@
 </work/11.ll>:fun3	no-nans-fp-math=false
 </work/11.ll>:fun3	norecurse
 </work/11.ll>:fun3	no-signed-zeros-fp-math=false
-</work/11.ll>:fun3	no-trapping-math=false
+</work/11.ll>:fun3	no-trapping-math=true
 </work/11.ll>:fun3	nounwind
 </work/11.ll>:fun3	readnone
 </work/11.ll>:fun3	stack-protector-buffer-size=8
@@ -63,7 +63,7 @@
 </work/11.ll>:fun4	no-nans-fp-math=false
 </work/11.ll>:fun4	norecurse
 </work/11.ll>:fun4	no-signed-zeros-fp-math=false
-</work/11.ll>:fun4	no-trapping-math=false
+</work/11.ll>:fun4	no-trapping-math=true
 </work/11.ll>:fun4	nounwind
 </work/11.ll>:fun4	readnone
 </work/11.ll>:fun4	stack-protector-buffer-size=8
@@ -82,7 +82,7 @@
 </work/11.ll>:id	no-nans-fp-math=false
 </work/11.ll>:id	norecurse
 </work/11.ll>:id	no-signed-zeros-fp-math=false
-</work/11.ll>:id	no-trapping-math=false
+</work/11.ll>:id	no-trapping-math=true
 </work/11.ll>:id	nounwind
 </work/11.ll>:id	readnone
 </work/11.ll>:id	stack-protector-buffer-size=8
@@ -101,7 +101,7 @@
 </work/11.ll>:main	no-jump-tables=false
 </work/11.ll>:main	no-nans-fp-math=false
 </work/11.ll>:main	no-signed-zeros-fp-math=false
-</work/11.ll>:main	no-trapping-math=false
+</work/11.ll>:main	no-trapping-math=true
 </work/11.ll>:main	nounwind
 </work/11.ll>:main	stack-protector-buffer-size=8
 </work/11.ll>:main	target-cpu=x86-64
@@ -117,7 +117,7 @@
 </work/11.ll>:malloc	no-infs-fp-math=false
 </work/11.ll>:malloc	no-nans-fp-math=false
 </work/11.ll>:malloc	no-signed-zeros-fp-math=false
-</work/11.ll>:malloc	no-trapping-math=false
+</work/11.ll>:malloc	no-trapping-math=true
 </work/11.ll>:malloc	nounwind
 </work/11.ll>:malloc	stack-protector-buffer-size=8
 </work/11.ll>:malloc	target-cpu=x86-64
--- 10/global_var_constant.csv.sorted	2022-10-25 15:45:37.816577820 -0400
+++ 11/global_var_constant.csv.sorted	2022-10-25 15:45:35.456560272 -0400
@@ -1,2 +1,2 @@
-</work/11.ll>:fun3:0:1:@global_obj
-</work/11.ll>:fun4:0:1:@global_obj
+</work/11.ll>:fun3:0:0:@global_obj
+</work/11.ll>:fun4:0:0:@global_obj
--- 10/global_var_constant_name.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/global_var_constant_name.csv.sorted	2022-10-25 15:45:35.457560279 -0400
@@ -1,2 +1,2 @@
-</work/11.ll>:fun3:0:1:@global_obj	@global_obj
-</work/11.ll>:fun4:0:1:@global_obj	@global_obj
+</work/11.ll>:fun3:0:0:@global_obj	@global_obj
+</work/11.ll>:fun4:0:0:@global_obj	@global_obj
--- 10/global_var_linkage.csv.sorted	2022-10-25 15:45:37.818577835 -0400
+++ 11/global_var_linkage.csv.sorted	2022-10-25 15:45:35.464560331 -0400
@@ -1 +1 @@
-</work/11.ll>:@global_obj	common
+</work/11.ll>:@global_obj	external
--- 10/instr_assigns_to.csv.sorted	2022-10-25 15:45:37.812577790 -0400
+++ 11/instr_assigns_to.csv.sorted	2022-10-25 15:45:35.499560591 -0400
@@ -1,11 +1,7 @@
 </work/11.ll>:fun1:0	</work/11.ll>:fun1:%call
 </work/11.ll>:fun1:1	</work/11.ll>:fun1:%0
-</work/11.ll>:fun1:2	</work/11.ll>:fun1:%call1
 </work/11.ll>:fun2:0	</work/11.ll>:fun2:%call
 </work/11.ll>:fun2:1	</work/11.ll>:fun2:%0
-</work/11.ll>:fun2:2	</work/11.ll>:fun2:%call1
-</work/11.ll>:fun3:0	</work/11.ll>:fun3:%call
-</work/11.ll>:fun4:0	</work/11.ll>:fun4:%call
 </work/11.ll>:main:0	</work/11.ll>:main:%call
 </work/11.ll>:main:10	</work/11.ll>:main:%2
 </work/11.ll>:main:11	</work/11.ll>:main:%add6
--- 10/instr_bb_entry.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/instr_bb_entry.csv.sorted	2022-10-25 15:45:35.501560607 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:0	</work/11.ll>:fun1:[basicblock]%entry
 </work/11.ll>:fun1:1	</work/11.ll>:fun1:[basicblock]%entry
 </work/11.ll>:fun1:2	</work/11.ll>:fun1:[basicblock]%entry
-</work/11.ll>:fun1:3	</work/11.ll>:fun1:[basicblock]%entry
 </work/11.ll>:fun2:0	</work/11.ll>:fun2:[basicblock]%entry
 </work/11.ll>:fun2:1	</work/11.ll>:fun2:[basicblock]%entry
 </work/11.ll>:fun2:2	</work/11.ll>:fun2:[basicblock]%entry
-</work/11.ll>:fun2:3	</work/11.ll>:fun2:[basicblock]%entry
 </work/11.ll>:fun3:0	</work/11.ll>:fun3:[basicblock]%entry
-</work/11.ll>:fun3:1	</work/11.ll>:fun3:[basicblock]%entry
 </work/11.ll>:fun4:0	</work/11.ll>:fun4:[basicblock]%entry
-</work/11.ll>:fun4:1	</work/11.ll>:fun4:[basicblock]%entry
 </work/11.ll>:id:0	</work/11.ll>:id:[basicblock]%entry
 </work/11.ll>:main:0	</work/11.ll>:main:[basicblock]%entry
 </work/11.ll>:main:10	</work/11.ll>:main:[basicblock]%entry
--- 10/instr_func.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/instr_func.csv.sorted	2022-10-25 15:45:35.504560629 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:0	</work/11.ll>:fun1
 </work/11.ll>:fun1:1	</work/11.ll>:fun1
 </work/11.ll>:fun1:2	</work/11.ll>:fun1
-</work/11.ll>:fun1:3	</work/11.ll>:fun1
 </work/11.ll>:fun2:0	</work/11.ll>:fun2
 </work/11.ll>:fun2:1	</work/11.ll>:fun2
 </work/11.ll>:fun2:2	</work/11.ll>:fun2
-</work/11.ll>:fun2:3	</work/11.ll>:fun2
 </work/11.ll>:fun3:0	</work/11.ll>:fun3
-</work/11.ll>:fun3:1	</work/11.ll>:fun3
 </work/11.ll>:fun4:0	</work/11.ll>:fun4
-</work/11.ll>:fun4:1	</work/11.ll>:fun4
 </work/11.ll>:id:0	</work/11.ll>:id
 </work/11.ll>:main:0	</work/11.ll>:main
 </work/11.ll>:main:10	</work/11.ll>:main
--- 10/instr_successor.csv.sorted	2022-10-25 15:45:37.812577790 -0400
+++ 11/instr_successor.csv.sorted	2022-10-25 15:45:35.506560644 -0400
@@ -1,11 +1,7 @@
 </work/11.ll>:fun1:0	</work/11.ll>:fun1:1
 </work/11.ll>:fun1:1	</work/11.ll>:fun1:2
-</work/11.ll>:fun1:2	</work/11.ll>:fun1:3
 </work/11.ll>:fun2:0	</work/11.ll>:fun2:1
 </work/11.ll>:fun2:1	</work/11.ll>:fun2:2
-</work/11.ll>:fun2:2	</work/11.ll>:fun2:3
-</work/11.ll>:fun3:0	</work/11.ll>:fun3:1
-</work/11.ll>:fun4:0	</work/11.ll>:fun4:1
 </work/11.ll>:main:0	</work/11.ll>:main:1
 </work/11.ll>:main:10	</work/11.ll>:main:11
 </work/11.ll>:main:11	</work/11.ll>:main:12
--- 10/pointer_type.csv.sorted	2022-10-25 15:45:37.817577828 -0400
+++ 11/pointer_type.csv.sorted	2022-10-25 15:45:35.567561098 -0400
@@ -3,4 +3,3 @@
 i8* (i64)*
 %struct.obj*
 %struct.obj* ()*
-%struct.obj* (%struct.obj*)*
--- 10/pointer_type_has_component.csv.sorted	2022-10-25 15:45:37.815577813 -0400
+++ 11/pointer_type_has_component.csv.sorted	2022-10-25 15:45:35.569561113 -0400
@@ -3,4 +3,3 @@
 i8*	i8
 %struct.obj*	%struct.obj
 %struct.obj* ()*	%struct.obj* ()
-%struct.obj* (%struct.obj*)*	%struct.obj* (%struct.obj*)
--- 10/ret_instr.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/ret_instr.csv.sorted	2022-10-25 15:45:35.583561217 -0400
@@ -1,6 +1,6 @@
-</work/11.ll>:fun1:3
-</work/11.ll>:fun2:3
-</work/11.ll>:fun3:1
-</work/11.ll>:fun4:1
+</work/11.ll>:fun1:2
+</work/11.ll>:fun2:2
+</work/11.ll>:fun3:0
+</work/11.ll>:fun4:0
 </work/11.ll>:id:0
 </work/11.ll>:main:15
--- 10/ret_instr_operand.csv.sorted	2022-10-25 15:45:37.820577850 -0400
+++ 11/ret_instr_operand.csv.sorted	2022-10-25 15:45:35.584561224 -0400
@@ -1,6 +1,6 @@
-</work/11.ll>:fun1:3	</work/11.ll>:fun1:%call1
-</work/11.ll>:fun2:3	</work/11.ll>:fun2:%call1
-</work/11.ll>:fun3:1	</work/11.ll>:fun3:%call
-</work/11.ll>:fun4:1	</work/11.ll>:fun4:%call
+</work/11.ll>:fun1:2	</work/11.ll>:fun1:%0
+</work/11.ll>:fun2:2	</work/11.ll>:fun2:%0
+</work/11.ll>:fun3:0	</work/11.ll>:fun3:0:0:@global_obj
+</work/11.ll>:fun4:0	</work/11.ll>:fun4:0:0:@global_obj
 </work/11.ll>:id:0	</work/11.ll>:id:%x
 </work/11.ll>:main:15	</work/11.ll>:main:%add8
--- 10/target_dependent_attr.csv.sorted	2022-10-25 15:45:37.816577820 -0400
+++ 11/target_dependent_attr.csv.sorted	2022-10-25 15:45:35.669561857 -0400
@@ -59,13 +59,13 @@
 no-signed-zeros-fp-math=false
 no-signed-zeros-fp-math=false
 no-signed-zeros-fp-math=false
-no-trapping-math=false
-no-trapping-math=false
-no-trapping-math=false
-no-trapping-math=false
-no-trapping-math=false
-no-trapping-math=false
-no-trapping-math=false
+no-trapping-math=true
+no-trapping-math=true
+no-trapping-math=true
+no-trapping-math=true
+no-trapping-math=true
+no-trapping-math=true
+no-trapping-math=true
 stack-protector-buffer-size=8
 stack-protector-buffer-size=8
 stack-protector-buffer-size=8
--- 10/type_has_size.csv.sorted	2022-10-25 15:45:37.815577813 -0400
+++ 11/type_has_size.csv.sorted	2022-10-25 15:45:35.675561901 -0400
@@ -7,4 +7,3 @@
 %struct.obj	8
 %struct.obj*	8
 %struct.obj* ()*	8
-%struct.obj* (%struct.obj*)*	8
--- 10/type_has_unpadded_size.csv.sorted	2022-10-25 15:45:37.813577797 -0400
+++ 11/type_has_unpadded_size.csv.sorted	2022-10-25 15:45:35.676561909 -0400
@@ -7,4 +7,3 @@
 %struct.obj	8
 %struct.obj*	8
 %struct.obj* ()*	8
-%struct.obj* (%struct.obj*)*	8
--- 10/variable.csv.sorted	2022-10-25 15:45:37.816577820 -0400
+++ 11/variable.csv.sorted	2022-10-25 15:45:35.698562072 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:%0
 </work/11.ll>:fun1:[basicblock]%entry
 </work/11.ll>:fun1:%call
-</work/11.ll>:fun1:%call1
 </work/11.ll>:fun2:%0
 </work/11.ll>:fun2:[basicblock]%entry
 </work/11.ll>:fun2:%call
-</work/11.ll>:fun2:%call1
 </work/11.ll>:fun3:[basicblock]%entry
-</work/11.ll>:fun3:%call
 </work/11.ll>:fun4:[basicblock]%entry
-</work/11.ll>:fun4:%call
 </work/11.ll>:id:[basicblock]%entry
 </work/11.ll>:id:%x
 </work/11.ll>:main:%0
--- 10/variable_has_name.csv.sorted	2022-10-25 15:45:37.819577842 -0400
+++ 11/variable_has_name.csv.sorted	2022-10-25 15:45:35.702562102 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:%0	%0
 </work/11.ll>:fun1:[basicblock]%entry	%entry
-</work/11.ll>:fun1:%call1	%call1
 </work/11.ll>:fun1:%call	%call
 </work/11.ll>:fun2:%0	%0
 </work/11.ll>:fun2:[basicblock]%entry	%entry
-</work/11.ll>:fun2:%call1	%call1
 </work/11.ll>:fun2:%call	%call
 </work/11.ll>:fun3:[basicblock]%entry	%entry
-</work/11.ll>:fun3:%call	%call
 </work/11.ll>:fun4:[basicblock]%entry	%entry
-</work/11.ll>:fun4:%call	%call
 </work/11.ll>:id:[basicblock]%entry	%entry
 </work/11.ll>:id:%x	%x
 </work/11.ll>:main:%0	%0
--- 10/variable_has_type.csv.sorted	2022-10-25 15:45:37.813577797 -0400
+++ 11/variable_has_type.csv.sorted	2022-10-25 15:45:35.703562109 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:%0	%struct.obj*
 </work/11.ll>:fun1:[basicblock]%entry	label
-</work/11.ll>:fun1:%call1	%struct.obj*
 </work/11.ll>:fun1:%call	i8*
 </work/11.ll>:fun2:%0	%struct.obj*
 </work/11.ll>:fun2:[basicblock]%entry	label
-</work/11.ll>:fun2:%call1	%struct.obj*
 </work/11.ll>:fun2:%call	i8*
 </work/11.ll>:fun3:[basicblock]%entry	label
-</work/11.ll>:fun3:%call	%struct.obj*
 </work/11.ll>:fun4:[basicblock]%entry	label
-</work/11.ll>:fun4:%call	%struct.obj*
 </work/11.ll>:id:[basicblock]%entry	label
 </work/11.ll>:id:%x	%struct.obj*
 </work/11.ll>:main:%0	i32
--- 10/variable_in_func_name.csv.sorted	2022-10-25 15:45:37.815577813 -0400
+++ 11/variable_in_func_name.csv.sorted	2022-10-25 15:45:35.704562117 -0400
@@ -1,15 +1,11 @@
 </work/11.ll>:fun1:%0	@fun1
 </work/11.ll>:fun1:[basicblock]%entry	fun1
-</work/11.ll>:fun1:%call1	@fun1
 </work/11.ll>:fun1:%call	@fun1
 </work/11.ll>:fun2:%0	@fun2
 </work/11.ll>:fun2:[basicblock]%entry	fun2
-</work/11.ll>:fun2:%call1	@fun2
 </work/11.ll>:fun2:%call	@fun2
 </work/11.ll>:fun3:[basicblock]%entry	fun3
-</work/11.ll>:fun3:%call	@fun3
 </work/11.ll>:fun4:[basicblock]%entry	fun4
-</work/11.ll>:fun4:%call	@fun4
 </work/11.ll>:id:[basicblock]%entry	id
 </work/11.ll>:id:%x	@id
 </work/11.ll>:main:%0	@main

```

</details>

Reply:
Looks like this is also #96